Bottom: 65ccfc7f8083eb908e78e35b47a0687160bbdb0d
Top:    b4ba466e367411215d4e04de30b2dc009aa31998
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-24 03:48:22 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 61eacba..8f698f4 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -283,6 +283,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
